src/tickable.ts

```diff
@@ -278,7 +278,7 @@ export abstract class Tickable extends Element {
    * @param mod the modifier
    */
   // eslint-disable-next-line
-  addModifier(index: number, mod: Modifier): this {
+  addModifier(mod: Modifier, index: number = 0): this {
```

Comment:
I believe that you can remove now `index` and the `eslint-disable...`

Reply:
DONE! This actually restores the Tickable API to be the same as version 3.0.9. Not sure when the unused index argument showed up.

3.0.9: https://unpkg.com/browse/vexflow@3.0.9/src/tickable.js

Reply:
Actually, I see now why we added the extra unused argument. After I made your suggested edit, the tests now throw 12 errors, because we call `getTickables()` and then call `addModifier()` on the tickable with an index argument. 

To fix this, we will have to cast each Tickable to Note.

I'd like to avoid forcing developers to add extra casts `(tickable as Note).addModifier(mod, index)`.

In this case, I think it's convenient to have Tickable have the same API as Note, but just ignore the second argument.